utils/test.py

```diff
@@ -0,0 +1,99 @@
+#
```

Comment:
I don't like having both `utils/test.py` and `utils/tests.py`

Reply:
Rename `utils/test.py` to `utils/test_helpers.py`.
Moved `utils/tests.py` to `tests` module inside `utils` module.
bc130f45c713bf47468afc54be2bab16c392b88a